bedrock/settings/base.py

```diff
@@ -1302,5 +1302,6 @@ def lazy_email_backend():
     'activist': '014783244707853607354:bbpl3emdsii',
 }
 
-if config('SWITCH_FLASHTALKING', default=DEV, cast=bool):
-    CSP_IMG_SRC += ('servedby.flashtalking.com',)
+if config('SWITCH_TRACKING_PIXEL', default=DEV, cast=bool):
+    CSP_IMG_SRC += ('ad.doubleclick.net',
```

Comment:
yeah... just can't import the `switch()` function here... so it's not quite as easy, but you can manually grab the switch environment variable. It's possible that I could make the switch function usable here, but I've not worked on that.